### DISCUSSION TITLE
Bi-directional entry relationship fieldtype

### DISCUSSION THREAD

**Discussion by samburgers** [statamic/cms#5914] [General]
Hi there, is Statamic able to provide many-to-many connections in the entry relationship field?

I can't find any info about that aside from this question from 2017

https://statamic.com/forum/2648-many-to-many-relationships

**Comment by duncanmcclean** [comment]
Many to Many relationships aren’t something that are natively supported by Statamic.

However, you could probably use an event listener that does the other ‘side’ of the relationship.

So if you had a Conference collection with Sponsor relations, you’d add a sponsor to the conference, then the listener would set the conference on the sponsor.

**Comment by samburgers** [comment]
Thanks for the quick reply @duncanmcclean ill give that a go, cheers 👍

**Comment by eliasu** [comment]
Hey @samburgers 
Any progress on that? Does it work? Could you give me a hint where to start?
Cheers!

**Reply by samburgers** [reply]
Hi @eliasu i haven't started yet, but i will fairly soon. this seems to be the right place to start however:

https://statamic.dev/extending/events#entrysaving

**Comment by samburgers** [comment]
@eliasu and anyone else that stumbles across this thread.. have just discovered this plugin which appears to do exactly this. 

https://statamic.com/addons/stillat/entry-relationships
